core/services/ocr2/plugins/ocr2keeper/evm20/encoder.go

```diff
@@ -0,0 +1,230 @@
+package evm
+
+import (
+	"fmt"
+	"math/big"
+	"reflect"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
+	"github.com/smartcontractkit/ocr2keepers/pkg/encoding"
+)
+
+type EVMAutomationEncoder20 struct {
+	encoding.BasicEncoder
+}
+
+var (
+	Uint256, _                = abi.NewType("uint256", "", nil)
+	Uint256Arr, _             = abi.NewType("uint256[]", "", nil)
+	PerformDataMarshalingArgs = []abi.ArgumentMarshaling{
+		{Name: "checkBlockNumber", Type: "uint32"},
+		{Name: "checkBlockhash", Type: "bytes32"},
+		{Name: "performData", Type: "bytes"},
+	}
+	PerformDataArr, _   = abi.NewType("tuple(uint32,bytes32,bytes)[]", "", PerformDataMarshalingArgs)
+	ErrUnexpectedResult = fmt.Errorf("unexpected result struct")
+	packFn              = reportArgs.Pack
+	unpackIntoMapFn     = reportArgs.UnpackIntoMap
+	mKeys               = []string{"fastGasWei", "linkNative", "upkeepIds", "wrappedPerformDatas"}
+	reportArgs          = abi.Arguments{
+		{Name: mKeys[0], Type: Uint256},
+		{Name: mKeys[1], Type: Uint256},
+		{Name: mKeys[2], Type: Uint256Arr},
+		{Name: mKeys[3], Type: PerformDataArr},
```

Comment:
this has changed, we added a triggers field and an execute gas field

Reply:
Thats in 2.0 folder